index.js

```diff
@@ -237,7 +237,7 @@ function checkInput(inputRequest, msg) {
   }
 
   // Display Position
-  if (inputRequest === "position") {
+  if (inputRequest.startsWith("position")) {
 
   leaderBoard.position(msg);
```

Comment:
Can you indent line 242 so it is clear it is in the `if` block

Reply:
Done in a new Pull Request. I don't know why it opened a new one